fastlane/lanes/release.rb

```diff
@@ -233,11 +244,10 @@
 
     # Check tags
     UI.user_error!("The version `#{new_version}` tag already exists!") if git_tag_exists(tag: new_version)
-    UI.user_error!("Version #{previous_version} is not tagged! A hotfix branch cannot be created.") unless git_tag_exists(tag: previous_version)
 
     # Create the hotfix branch
-    UI.message 'Creating hotfix branch...'
-    Fastlane::Helper::GitHelper.create_branch("release/#{new_version}", from: previous_version)
+    UI.message "Creating hotfix branch from #{base_ref_for_hotfix}..."
```

Comment:
As per https://github.com/wordpress-mobile/WordPress-iOS/pull/24996/files#r2535967743

```suggestion
    UI.message("Creating hotfix branch from #{base_ref_for_hotfix}...")
```

Reply:
👍 https://github.com/wordpress-mobile/WordPress-Android/commit/af7345511bb44e246d884597ed1806d68627eef5